### PR TITLE
Make Bioswale participate in counts

### LINKED ISSUE #2110: Bioswales not included in resource counts
Reproducible on staging.

1) Create a non-bioswale resource
2) watch the count go up
3) Create a bioswale
4) Count does not go up

### PR DESCRIPTION
**Pull request by steventlamb**
fixes #2110 on github

### CODE REVIEW

File: opentreemap/treemap/ecobenefits.py [hunk 1]
```diff
@@ -80,6 +80,21 @@ def benefits_for_object(self, instance, obj):
...
+class CountOnlyBenefitCalculator(BenefitCalculator):
+    def __init__(self, clz):
+        self.clz = clz
+
+    def benefits_for_filter(self, instance, item_filter):
+        features = item_filter.get_objects(self.clz)
+        return ({},
+                {'resource':
+                 {'n_objects_used': 0,
+                  'n_objects_discarded': len(features)}})
```
Comment (maurizi) [inline opentreemap/treemap/ecobenefits.py]: [`get_objects`](https://github.com/OpenTreeMap/otm-core/blob/4f6c657f2337d766a10268d9d422de7e082c22e8/opentreemap/treemap/search.py#L54-L84) returns a queryset, so I think you can change `len(features)` here to be `features.count()`.

**Comment by maurizi** [comment]
I really like how this turned out, with `CountOnlyBenefitCalculator` being reuseable for different `MapFeature` models.  Will test shortly.

**Comment by maurizi** [comment]
+1, works well.

**Comment by steventlamb** [comment]
@maurizi change made, and pushed some other tweaks to the markup/presentation

**Comment by maurizi** [comment]
Still +1

**Pull request merged**